### PR TITLE
Merge RC3 into rel/1.0.0

### PR DESCRIPTION
**Pull request by krwq**
cc: @piotrpMSFT @livarcocc 

take a closer look at build\Microsoft.DotNet.Cli.DependencyVersions.props

### CODE REVIEW

File: build/Microsoft.DotNet.Cli.DependencyVersions.props [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CLI_MSBuild_Version>15.1.0-preview-000522-02</CLI_MSBuild_Version>
```
Comment (krwq) [inline build/Microsoft.DotNet.Cli.DependencyVersions.props]: I took the highest versions here

Comment (livarcocc) [inline build/Microsoft.DotNet.Cli.DependencyVersions.props]: Should be all right.

**Comment by livarcocc** [comment]
@dotnet-bot Test Ubuntu16.04 x64 Debug Build

**Comment by livarcocc** [comment]
@MattGertz for approval.

**Pull request merged**